.github/workflows/style.yml

```diff
@@ -1,28 +1,31 @@
+---
```

Comment:
Any reason to add the split here? (same for the other workflows) 

```suggestion
```

Reply:
yamllint *.yml

`warning  missing document start "---"`